storage/objects/generate_signed_post_policy_v4.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objects
+
+// [START storage_generate_signed_post_policy_v4]
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"io/ioutil"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/oauth2/google"
+)
+
+// generateSignedPostPolicyV4 generates a signed post policy.
+func generateSignedPostPolicyV4(w io.Writer, bucket, object, serviceAccountJSONPath string) (*storage.PostPolicyV4, error) {
```

Comment:
Minor nit.
Perhaps rename “w” to “wForm”

Reply:
All the other samples in this directory seem to use w for the writer that's passed in, so I think I'll leave this.